src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const daysCountMinDiscount = 3;
+  const daysCountMaxDiscount = 7;
+
+  if (days < daysCountMinDiscount) {
+    return rentCost * days;
+  }
+
+  if (days >= daysCountMinDiscount && days < daysCountMaxDiscount) {
+    return rentCost * days - minDiscount;
+  }
+
+  if (days >= daysCountMaxDiscount) {
+    return rentCost * days - maxDiscount;
+  }
```

Comment:
Don't duplicate conditions. If previous condition is `false`, the opposite is always `true`.
And last condition is useless in this case.

```suggestion
  if (days < daysCountMaxDiscount) {
    return rentCost * days - minDiscount;
  }
    
  return rentCost * days - maxDiscount;
```